web-page/src/components/FilterSideBar/RarityFilter/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import FilterButtonsRow from '../FilterButtonsRow'
+import { attributes as CardAttributes} from '../../../cards.json'
+
+const CARD_RARITY_TYPES = CardAttributes.rarity.map(
+    item => {
+        const match = item.match(/.*\[(?<rarity>.+)\]/)
+        if (!match) {
+            return { label: item, value: item}
+        }
+        
+        const { groups: { rarity } } = match
+        return { label: rarity, value: item}
```

Comment:
Add a new line before a return.
```suggestion

        return { label: rarity, value: item}
```